yapf/__init__.py

```diff
@@ -30,6 +30,7 @@
 import argparse
 import logging
 import sys
+from lib2to3.pgen2.parse import ParseError
```

Comment:
Ooops. Will push a fix.

Originally I was thinking I needed to catching ParseErrors too (it's not clear to me if/when they would ever happen; i.e. if ParseError => SyntaxError). I don't think they need to be caught.